src/templates/globals.mustache.js

```diff
@@ -56,16 +59,23 @@ window.QUnitGlobals = {
                 start();
             }, WAIT_FOR_IFRAME_TIMEOUT);
 
-            intervalId = window.setInterval(function () {
-                if (iframe && iframe.contentWindow) {
-                    window.clearInterval(intervalId);
+            try {
+                isIframeLoaded = iframe.contentWindow && iframe.contentWindow.document.readyState === 'complete';
+            }
+            catch (e) {
+                //NOTE: if cross-domain iframe raises an error we take it as loaded
+                isIframeLoaded = true
+            }
 
-                    iframe.contentWindow.addEventListener('load', function () {
-                        window.clearTimeout(timeoutId);
-                        resolve();
-                    });
-                }
-            }, CHECK_PREDICATE_INTERVAL);
+            if (isIframeLoaded) {
+                window.clearTimeout(timeoutId);
+                resolve();
```

Comment:
`return;` requires. We should not add a listener to the `load` event if the iframe is already loaded